File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/statistikk/sykefraværshistorikk/aggregert/StatistikkDto.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.sykefraværshistorikk.aggregert;

import java.util.List;
import lombok.Builder;
import lombok.Data;
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.felles.ÅrstallOgKvartal;
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.Statistikkategori;

@Data
@Builder
public class StatistikkDto {

  private Statistikkategori statistikkategori;
  private String label;
  private String verdi;
  private Integer antallPersonerIBeregningen;
  private List<ÅrstallOgKvartal> kvartalerIBeregningen;
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/statistikk/sykefraværshistorikk/aggregert/StatistikkDto.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.sykefraværshistorikk.aggregert

import no.nav.arbeidsgiver.sykefravarsstatistikk.api.felles.ÅrstallOgKvartal
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.Statistikkategori

data class StatistikkDto(
    val statistikkategori: Statistikkategori? = null,
    val label: String? = null,
    val verdi: String? = null,
    val antallPersonerIBeregningen: Int? = null,
    val kvartalerIBeregningen: List<ÅrstallOgKvartal>? = null
)